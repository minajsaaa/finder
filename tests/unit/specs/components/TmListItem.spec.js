import { mount } from "@vue/test-utils";
import TmListItem from "@/components/TmListItem";

describe(`TmListItem`, () => {
  let wrapper, localThis;
  beforeEach(() => {
    wrapper = mount(TmListItem);
  });

  it(`TmListItem Snapshot`, () => {
    expect(wrapper.vm.$el).toMatchSnapshot();
  });

  describe(`Computed`, () => {
    beforeEach(() => {
      localThis = {
        spin: true
      };
    });

    it(`TmListItem`, () => {
      expect(TmListItem.computed.spinClass.call(localThis)).toBe("fa-spin");
    });

    it(`TmListItem`, () => {
      expect(TmListItem.computed.spinClass()).toBe(null);
    });
  });
});
