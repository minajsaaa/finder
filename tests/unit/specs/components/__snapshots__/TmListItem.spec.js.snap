// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TmListItem TmListItem Snapshot 1`] = `<!---->`;
